### PR TITLE
UI Changes

### PR DESCRIPTION
**Pull request by tiffchou**
Changes have been made to both the UI and some back-end functionality. Resizing capabilities have been added to most components. Multiple items can now be deleted at the same time on the initial Profiles page. Also, there was an error that would occur when the user attempted to check off the "Response"/"Request" column immediately after creating a new Path on the Edit Profile page, but it's fixed.

Some other minor changes to the UI have been made.

### CODE REVIEW

**Comment by dwillson** [comment]
You have some files included that should be removed from the repo:
.DS_Store
backup-uploaded.json
h2proxydb/*
proxyui/odo.war
proxyui/plugins/plugins-jar-with-dependencies.jar
proxyui/seleniumSslSupport/*

**Comment by dwillson** [comment]
+proxyui/tomcat.ks

File: proxylib/src/main/java/com/groupon/odo/proxylib/ClientService.java [hunk 1]
```diff
@@ -144,9 +144,11 @@ public Client getClient(int clientId) throws Exception {
      * @throws Exception
      */
     public Client findClient(String clientUUID, Integer profileId) throws Exception {
+        logger.info("HELLOOOOOOOOOOOOOOOOOOOOooooooš", "TIFFANY");
```
Comment (dwillson) [inline proxylib/src/main/java/com/groupon/odo/proxylib/ClientService.java]: remove debug

Comment (tiffchou) [inline proxylib/src/main/java/com/groupon/odo/proxylib/ClientService.java]: Missed that. Thanks!

File: proxyui/src/main/webapp/WEB-INF/views/profiles.jsp [hunk 2]
```diff
@@ -36,11 +37,14 @@
             return cellvalue;
         }
 
-        // formater for the name column
+        // formatter for the name column
         function nameFormatter( cellvalue, options, rowObject ) {
-            var cellContents = '<div class="ui-state-default ui-corner-all" style="float:right" title="Edit Profile" onClick="editProfile(' + currentProfileId + ')">';
+            /*var cellContents = '<div class="ui-state-default ui-corner-all" style="float:right" title="Edit Profile" onClick="editProfile(' + currentProfileId + ')">';
```
Comment (dwillson) [inline proxyui/src/main/webapp/WEB-INF/views/profiles.jsp]: delete commented code

File: proxyui/src/main/webapp/WEB-INF/views/profiles.jsp [hunk 3]
```diff
@@ -95,10 +104,14 @@
...
                 }, {
                     name : 'name',
-                    index : 'displayProfileName',
+                    index : 'name',//'displayProfileName',
                     width : 400,
                     editable : false,
-                    formatter: nameFormatter
+                    formatter: nameFormatter,
+                    sortable:true,
+                    //sorttype:function(){
```
Comment (dwillson) [inline proxyui/src/main/webapp/WEB-INF/views/profiles.jsp]: remove commented code

**Comment by dwillson** [comment]
Lots of comments in GenericTestProxyContainer.java, HttpProxyContainer.java, can remove those that are extraneous in release code.

Looked through edit.jsp, lots of diff due to formatting update. But seems ok, just double check for commented code that should be removed.

**Pull request closed without merging**